### PR TITLE
mention int8 and bfloat16 cell types

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@havardpe please review

### CODE REVIEW

**Review by havardpe — APPROVED** [review]
looks good

maybe drop saying that we will automatically cast to 32-bit float (results of calculations will be as if it was a 32-bit float)

(either) maybe refer to cell_cast operation as a way to ensure tensors have the same cell type to enable back-end optimization.
(or) probably better: we add the needed cell_casts automatically in the back-end.

File: en/tensor-user-guide.html [hunk 1]
```diff
@@ -168,13 +168,48 @@ <h2 id="performance-considerations">Performance considerations</h2>
 Even though you would typically not do the above without reducing the
 search space first (using matching and first phase), it is important to consider
 the memory bandwidth and other hardware limitations when developing ranking expressions with tensors.
-</p><p>
+</p>
+<h3 id="cell-value-types">Performance considerations for cell value types</h2>
```
Comment (havardpe) [inline en/tensor-user-guide.html:172]: header mismatch h2 vs h3

**Review by havardpe — APPROVED** [review]
looks good

**Comment by arnej27959** [comment]
@kkraune please take a look

**Review by kkraune — APPROVED** [review]
Great!

**Pull request merged**